cmd/sedad/utils/merkle.go

```diff
@@ -0,0 +1,107 @@
+package utils
+
+import (
+	"bytes"
+	"hash"
+	"math/bits"
+
+	"golang.org/x/crypto/sha3"
+)
+
+// RootFromEntries computes the root of the Merkle Hash Tree whose
+// leaves are the entries given in byte slices in the provided order.
+// It largely follows RFC-6962 with some modifications:
+//   - It uses SHA3-256 hashing algorithm.
```

Comment:
Is this still true?

Reply:
ah right. changed "SHA3-256" to "Keccak-256"